src/components/sound-picker/sound-picker.tsx

```diff
@@ -7,17 +7,36 @@ import LabelsIcon from "../../assets/icons/sell_black_48dp.svg";
 import "./sound-picker.scss";
 
 export interface ISoundPickerProps {
-  selectedSound: SoundName
+  selectedSound: SoundName;
+  drawWaveLabels: boolean;
   handleSoundChange?: (event: ChangeEvent<HTMLSelectElement>) => void;
   onRecordingCompleted?: (audioBuffer: AudioBuffer) => void;
+  handleDrawWaveLabelChange?: () => void;
 }
 
+export const isPureTone = (soundName: string) => {
+  switch (soundName) {
+    case "middle-c": return true; break;
+    case "c2": return true; break;
```

Comment:
`break` is unnecessary after `return`. I'm surprised that's not generating unreachable code warnings.

Reply:
FYI: some linters have rules that require a `break;` for each `case` (presumably to avoid un-intended fall-through bugs). But I'll change the code; as suggested.

Reply:
Such [no-fallthrough](https://eslint.org/docs/rules/no-fallthrough) rules generally accept `break` or `return`. It would be pointless to require a `break` _after_ a `return`.